meinberlin/apps/plans/views.py

```diff
@@ -35,7 +35,7 @@ class PlanListView(rules_mixins.PermissionRequiredMixin,
     permission_required = 'meinberlin_plans.list_plan'
 
     def get_queryset(self):
-        return super().get_queryset()\
+        return super().get_queryset() \
```

Comment:
There shouldn't be a blank inbetween, because it all goes together `super().get_queryset().select_related()`
Same below in lines 50-52